src/Containers/HighlightedText/HighlightedText.tsx

```diff
@@ -41,13 +45,21 @@ export const HighlightedText: React.FC<HighlightedTextProps> = ({
      * construct the element for a special text
      * @param label - HighlightedString of the special text
      */
-    const getSpecialTextComponent = (label: HighlightedString): React.ReactElement => <ClickableSmallText bold {...label.textProps}>{`${label.text  } `}</ClickableSmallText>
+    const getSpecialTextComponent = (label: HighlightedString): React.ReactElement => {
+        const [textColor, setTextColor] = useState('black');
+        const handleChangeTextColor = () => {
+            setTextColor(textColor === 'black' ? 'orange' : 'black');
+        }
```

Comment:
Since state is passed as a prop to a child component ClickableSmallText is it still possible to use Styled Components to apply the CSS to ClickableSmallText?